src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,59 @@
+package duke.task;
+
+/**
+ * Represents a Task created by the user.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Default constructor for a Task.
+     *
+     * @param description Description of task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
```

Comment:
Is the overqualified 'this' necessary? https://se-education.org/guides/conventions/java/index.html under variables